comp/core/tagger/README.md

```diff
@@ -62,31 +48,29 @@ cache. Cache invalidation is triggered by the collectors (or source) by either:
 Tagger entities are identified by a string-typed ID, with one of the following forms:
 
 <!-- NOTE: a similar table appears in comp/core/autodiscovery/README.md; please keep both in sync -->
-| *Service*                               | *Tagger Entity*                                                    |
-|-----------------------------------------|--------------------------------------------------------------------|
-| workloadmeta.KindContainer              | `container_id://<sha>`                                             |
-| workloadmeta.KindContainerImageMetadata | `container_image_metadata://<sha>`                                 |
-| workloadmeta.KindGardenContainer        | `container_id://<sha>`                                             |
-| workloadmeta.KindKubernetesPod          | `kubernetes_pod_uid://<uid>`                                       |
-| workloadmeta.KindECSTask                | `ecs_task://<task-id>`                                             |
-| CloudFoundry LRP                        | `<processGuid>/<svcName>/<instanceGuid>`  or `<appGuid>/<svcName>` |
-| Container runtime or orchestrator       | (none)                                                             |
-| Kubernetes Endpoint                     | `kube_endpoint_uid://<namespace>/<name>/<ip>`                      |
-| Kubernetes Service                      | `kube_service://<namespace>/<name>`                                |
```

Comment:
why did some services disappear? were they removed from the code but not from documentation?

Reply:
I think so. Here's the current list of the objects handled by the tagger: https://github.com/DataDog/datadog-agent/blob/854f00295eab22f9d07851a66bfd847cc8eefc2e/comp/core/tagger/taggerimpl/collectors/workloadmeta_extract.go#L132 All of them come from workloadmeta